app/Models/Server.php

```
@@ -171,6 +171,24 @@ protected static function booted()
                 $server->proxy->redirect_enabled = true;
             }
+
+            // Create predefined server shared variables
+            SharedEnvironmentVariable::create([
+                'key' => 'COOLIFY_SERVER_UUID',
+                'value' => $server->uuid,
+                'type' => 'server',
+                'server_id' => $server->id,
+                'team_id' => $server->team_id,
+                'is_literal' => true,
+            ]);
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Models\SharedEnvironmentVariable' in method 'booted'. (undefined)

(StaticAccess)